cli/valet.php

```diff
@@ -234,38 +234,82 @@
     /**
      * Start the daemon services.
      */
-    $app->command('start', function () {
-        DnsMasq::restart();
-
-        PhpFpm::restart();
-
-        Nginx::restart();
+    $app->command('start [service]', function ($service) {
+        switch($service) {
+            case '':
+                DnsMasq::restart();
+                PhpFpm::restart();
+                Nginx::restart();
+
+                info('Valet services have been started.');
```

Comment:
Can re modify all of  these to `return` the message? 

`return info('Valet services have been started.');`

Just helpful because it drops an extra line.

Reply:
Can you also make sure that you have a newline before each of these switch return statements?